pkg/appstore/AppStoreDeploymentService.go

```diff
@@ -1320,7 +1320,7 @@ func (impl *InstalledAppServiceImpl) Subscribe() error {
 				impl.logger.Errorw(" error", "err", err)
 			}*/
 		}
-	}, stan.DurableName(BULK_APPSTORE_DEPLOY_DURABLE), stan.StartWithLastReceived(), stan.AckWait(time.Duration(180)*time.Second), stan.SetManualAckMode(), stan.MaxInflight(1))
+	}, stan.DurableName(BULK_APPSTORE_DEPLOY_DURABLE), stan.StartWithLastReceived(), stan.AckWait(time.Duration(200)*time.Second), stan.SetManualAckMode(), stan.MaxInflight(3))
```

Comment:
Why did we change it? was there delay in ack? also not processing can be out of order, is there any impact?

Reply:
sometimes msg gets stuck in pending state, root cause is yet not known. This workaround will unblock user for default cluster component installations.
out of order msg doesn't have any impact.